TODO-rebase.sh

```diff
@@ -206,26 +206,26 @@ run_specific_rebase () {
 		export GIT_EDITOR
 		autosquash=
 	fi
-	. git-rebase--$type
+	# . git-rebase--$type
 
-	if test -z "$preserve_merges"
-	then
-		git_rebase__$type
+	# if test -z "$preserve_merges"
+	# then
+	# 	git_rebase__$type
 	else
 		git_rebase__preserve_merges
```

Comment:
But you already have that in `builtin-rebase.c`... Should this not be commented-out, then?